src/api/cart.js

```diff
@@ -11,6 +11,10 @@ export default class extends Base {
     getCart(options = {}, callback) {
         let url = '/api/storefront/cart';
 
+        if (options.cartId) {
+            url = `/api/storefront/carts/${options.cartId}`;
+        }
+
```

Comment:
Not necessary, nor sure if it makes it cleaner but you could also use the ternary operator here:

`url = options.cartId ? `/api/storefront/carts/${options.cartId}` : `/api/storefront/carts/``